### ISSUE TITLE
Directory Listing Only Lists Single Directory & Ignores Regular Requests

### ISSUE THREAD

**Issue by trisys3** [BrowserSync/browser-sync#1313]
### Issue details

_Please provide issue details here_.

When using multiple base directories with the `server.baseDir` option, then using the `directory` option, only the 1st base directory is listed. In addition, files from the other directories are not returned when requested.

For example, with the simplified configuration listed below, if I request `http://my.awesome.host` in my favorite browser, I get the directory listing for the `app` folder inside the current working directory. This listing shows only files and folders from `app`, and none from `temp`. Also, if `app.css` is inside the `app` folder and `app.js` is inside the `temp` folder, I can reach `app.css` by navigating to `http://my.awesome.host/app.css`, but I can not reach `app.js`.

### Steps to reproduce/test case

_Please provide necessary steps for reproduction of this issue, or better the
reduced test case (without any external dependencies)_.

Configuration:

    const bsConf = {
        host: 'my.awesome.host',
        server: {
            directory: true,
            baseDir: [
                // at least 2 directories, e.g.
                'app',
                'temp',
            ],
        },
    }

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.18.6  ]
- Node        [  4.7.2  ]
- Npm         [  3.10.10  ]

### Affected platforms

- [x ] linux
- [ ] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ x ] Gulp
- [ ] Grunt
- [ ] CLI

### If CLI, please paste the entire command below

{cli command here}

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

{Browsersync init code here}

Simplified `gulpfile.js` (really using multiple files, etc.):

'use strict';

// as above
const bsConf = {
    server: {
        directory: true,
        baseDir: [
            'app',
            'temp',
        ],
    },
};

gulp.task('serve', () => {
    const pkg = require(`./package.json');
    const bs = require('browser-sync').create(pkg.name);

    bs.init(bsConf);
});

**Comment by shakyShane** [comment]
@trisys3 yeah I don't recall ever supporting multiple paths for the `directory` option unfortunately!

**Comment by trisys3** [comment]
That's fine, I really only added it temporarily to test a new route I wasn't sure would work. It was helpful, and would have been *slightly* more helpful if it worked with more than one base directory, but only slightly. I was ultimately able to make do.
